mec/pkg/pullstrategy/ossm/strategy.go

```diff
@@ -72,9 +71,6 @@ func (p *ossmPullStrategy) createImageStreamImport(imageStreamName string, image
 						Kind: "DockerImage",
 						Name: image.String(),
 					},
-					To: &corev1.LocalObjectReference{
```

Comment:
why did this have to be removed?

Reply:
This field is meant to host the tag, not the image name. If omitted, it defaults to the tag, which is what we want, and is less confusing.

Reply:
gotcha, thanks